harvard-lts/CURIOSity
# Upgrades bundler and system gems so we dont have to lock uri anymore

**Upgrades bundler and system gems**
* * *

# What does this Pull Request do?
This will gt us on bundler 2.4.5 and upgrade the pre-packed uri gem so we don't get that mismatch error message that forced us to lock to uri 0.10 before.

# How should this be tested?

Spin up this branch and make sure the site still works. You can also shell in and run bundle install and then a db:migrate and both should work without that error (in the Gemfile.lock you can see uri is 0.12 now)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
